pkg/cmd/cli/cmd/helper.go

```diff
@@ -23,23 +23,25 @@ func selectContainers(containers []kapi.Container, spec string) ([]*kapi.Contain
 }
 
 func handlePodUpdateError(out io.Writer, err error, resource string) {
```

Comment:
I think part of the problem is the design of CheckErr.  We should ideally
invert it (on the factory, not a static method), then wrap it on a per
command case with variants that handle specific error cases.  I don't know
if we have an existing follow up for this.

On Mon, Aug 10, 2015 at 10:21 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/helper.go
> <https://github.com/openshift/origin/pull/4055#discussion_r36635837>:
>
> > @@ -23,23 +23,25 @@ func selectContainers(containers []kapi.Container, spec string) ([]*kapi.Contain
> >  }
> >
> >  func handlePodUpdateError(out io.Writer, err error, resource string) {
>
> I know this is prexisting, but if we want something like this made pretty,
> why don't we have a generic translation is CheckErr?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4055/files#r36635837>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift